modules/network-observability-flowmetric-api-specifications.adoc

```diff
@@ -120,6 +120,10 @@ Refer to the documentation for the list of available fields: https://docs.opensh
 | `string`
 | Name of the metric. In Prometheus, it is automatically prefixed with "netobserv_".
 
+| `remap`
+| `object (string)`
+| `remap` allows to use different names for the generated metric labels than the flow fields. Use the origin flow fields as keys, and the desired label names as values.
```

Comment:
```suggestion
| Set the `remap` property to use different names for the generated metric labels than the flow fields. Use the origin flow fields as keys, and the label names that you want as values.
```
- I reworded this line a little to avoid anthropomorphic language ("allows").
- IBM Style word usage warns against using "desired" in technical content.